ui/src/app/components/direct-connection/direct-connection.component.ts

```diff
@@ -48,8 +54,8 @@ export class DirectConnectionComponent implements OnInit {
     window.scroll(0, 0)
     this.data.resetStore()
     localStorage.clear()
-    const { dbEngine, hostName, port, userName, password, dbName } = this.connectForm.value
-    const config: IDbConfig = { dbEngine, hostName, port, userName, password, dbName }
+    const { dbEngine, hostName, port, userName, password, dbName, dialect } = this.connectForm.value
+    const config: IDbConfig = { dbEngine, hostName, port, userName, password, dbName, dialect }
```

Comment:
Having `dialect` here is misleading since this reads as "dialect is relevant to the source database connection". We are deriving this from `connectForm` which is for source db connection information and storing it in `IDbConfig` which I again assume to "Input" Db config. This sounds wrong.

Reply:
Moved dialect out of IDbConfig